instance/tasks.py

```diff
@@ -92,16 +91,28 @@ def spawn_appserver(
 def make_appserver_active(appserver_id, active=True, deactivate_others=False):
     """
     Mark an AppServer as active or inactive.
+
+    :returns True if successful, False otherwise.
```

Comment:
I would clarify this comment a bit more to be explicit about what successful really means in this function's context.